documentation/source/user_section/tutorials/4d/processing-dmri-data/motion-correction-for-dmri.rst

```diff
@@ -0,0 +1,35 @@
+Motion correction for dMRI images
+#################################
+
+SCT features a complex motion correction algorithm, which is inspired by `[Xu et al., Neuroimage 2013] <https://pubmed.ncbi.nlm.nih.gov/23178538/>`_.
+
+.. TODO: The explanation of the motion correction algorithm and the specific parameters is a little confusing to me. Should we rewrite into a step-by-step process? Make it more brief? Leave some of the explanation for the ``-h`` description?
+
+   The key aspects of this algorithm are as follows:
+
+    * **SliceReg:** Slice-wise registration regularized along the Z direction (based on the function antsSliceRegularizedRegistration from ANTs, and described in [De Leener et al., Neuroimage 2017]).
+    * **Grouping:** there is the possibility to group successive volumes in order to have sufficient SNR to estimate a reliable transformation. If your data are very low SNR you can increase the number of successive images that are averaged into group with the flag ``-g``.
+    * **Iterative average:** after registering a new group to the target image (which is usually the first DWI group), the target is averaged with the newly registered group in order to increase the SNR of the target image.
+    * **Outlier detection:** if a detected transformation is too large, it is ignored and the previous transformation is used instead.
+    * **Masking:** in order to estimate motion of the cord, ignoring the rest of the tissue, there is the possibility to include a mask with the flag ``-m``.
+
+.. code::
+
+   sct_dmri_moco -i dmri_crop.nii.gz -bvec bvecs.txt -qc ~/qc_singleSubj -qc-seg TODO
+
+:Input arguments:
+   - ``-i`` : The input dMRI image.
+   - ``-bvec`` : A text file with three lines, each containing a value for each volume in the input image. Together, the the three sets of values represent the ``(x, y, z)`` coordinates of the b-vectors, which indicate the direction of the diffusion encoding for each volume of the dMRI image.
+   - ``-qc`` : Directory for Quality Control reporting. QC reports allow us to evaluate the results slice-by-slice.
+   - ``-qc-seg`` : TODO: QC requires a segmentation. However, the segmentation produced in the previous "Preprocessing" page was not cropped, so, it doesn't work with ``-qc-seg``! We will need to rewrite either the tutorial or the QC code so that they work together to form a "typical" usage.
```

Comment:
The current `sct_dmri_moco` QC usage isn't compatible with this tutorial, as the preprocessing doesn't generate a suitable segmentation file. :sweat_smile: 

I'm wondering... could we rewrite the QC to run `deepseg_sc` internally on the `dwi_mean` file? That way, we wouldn't require a separate seg file at all, avoiding having to do 2 passes.

Reply:
Great idea! Works perfectly. 

![Screenshot from 2021-08-12 11-21-32](https://user-images.githubusercontent.com/16181459/129223382-04725c25-e4b8-4f90-abfb-23c5b59087b0.png)